http-server/src/main/java/org/triplea/server/http/spark/controller/SparkController.java

```diff
@@ -0,0 +1,7 @@
+package org.triplea.server.http.spark.controller;
+
+/**
+ * Marker interface for any spark component that will register an endpoint and associated handler.
+ */
+public interface SparkController extends Runnable {
```

Comment:
Isn't this interface really unecessary

Reply:
269006c90 removes the unnecessary "aliasing" interface. I'm compromising on this style in another project that it got a bit too ingrained :worried: